.github/workflows/codeql.yml

```diff
@@ -40,8 +44,24 @@ jobs:
           build-mode: manual
       - name: Install Taskfile support
         uses: arduino/setup-task@v2
-      - shell: bash
-        run: task build
+      - name: Build (CodeQL-instrumented)
+        shell: bash
+        env:
+          GOTOOLCHAIN: local
+        run: |
+          # TODO(GODRIVER-3723): Run using taskfile targets.
+          go build ./...
```

Comment:
[blocking] We need to include the libmongocrypt task before building:

```
task install-libmongocrypt
```

If this doesn't work, try running the script manually:

```
bash etc/install-libmongocrypt.sh
test -d install || test -d /cygdrive/c/libmongocrypt/bin
```

If we have to do the latter, we should remove taskfile support:

```
      - name: Install Taskfile support
        uses: arduino/setup-task@v2
```